### PR TITLE
Redirect to docs.brew.sh

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Let's redirect this to https://docs.brew.sh/Linuxbrew instead.

CC @andrew 

Part of https://github.com/Homebrew/brew/issues/5835

### CODE REVIEW

**Comment by sjackman** [comment]
PR https://github.com/Homebrew/brew.sh/pull/337 moves the deleted file `linuxbrew-256x256.png` to `brew.sh/assets/img`.

**Review by sjackman — COMMENTED** [review]
Where is the destination `https://docs.brew.sh/Linuxbrew` of the redirect specified?

**Comment by sjackman** [comment]
PR https://github.com/Homebrew/brew.sh/pull/337 and
PR https://github.com/Homebrew/brew/pull/5870
must be merged before this PR.

**Comment by MikeMcQuaid** [comment]
> Where is the destination `https://docs.brew.sh/Linuxbrew` of the redirect specified?

@sjackman Forgot to add the `index.html`; done now!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Homebrew on Linux
+redirect_to: https://docs.brew.sh/Linuxbrew
```
Comment (sjackman) [inline index.html]:
```suggestion
redirect_to: https://docs.brew.sh/Linux
```

**Review by sjackman — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (MikeMcQuaid) [reply index.html]: That page does not exist.

Comment (sjackman) [reply index.html]: See https://github.com/Homebrew/brew/pull/5867#issuecomment-472125846

Comment (MikeMcQuaid) [reply index.html]: It will be `/Homebrew-on-Linux-and-Windows-10` once https://github.com/Homebrew/brew/pull/5867/ is merged.

**Review by sjackman — APPROVED** [review]
Thanks, Mike!

**Pull request merged**